chartfx-dataset/src/main/java/de/gsi/dataset/serializer/spi/AbstractSerialiser.java

```diff
@@ -149,32 +139,23 @@ public void serialiseObject(final Object obj) throws IllegalAccessException {
         serialiseObject(obj, ClassDescriptions.get(obj.getClass()), 0);
     }
 
-    public void serialiseObject(final Object obj, final ClassFieldDescription root, final int recursionDepth)
+    public void serialiseObject(final Object rootObj, final ClassFieldDescription root, final int recursionDepth)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'serialiseObject(Object, ClassFieldDescription, int)' has an NPath complexity of 432](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5640920)

Reply:
complexity remains unchanged compared to previous version (Codacy probably did not detect continuity due to reformatting)